autoload/fzf/vim.vim

```diff
@@ -618,9 +618,9 @@ function! fzf#vim#gitfiles(args, ...)
   let prefix = 'git -C ' . fzf#shellescape(root) . ' '
   if a:args != '?'
     return s:fzf('gfiles', {
-    \ 'source':  prefix . 'ls-files '.a:args.(s:is_win ? '' : ' | uniq'),
+    \ 'source':  prefix . 'ls-files -z'.a:args.(s:is_win ? '' : ' | uniq'),
```

Comment:
Are you running this on Windows? `uniq` here cannot properly handle NUL character terminated list.

Reply:
Related: https://github.com/junegunn/fzf.vim/pull/473

Reply:
Ah we can use `--deduplicate`.

```suggestion
    \ 'source':  prefix . 'ls-files -z --deduplicate',
```